### PR TITLE
KFSTI-1776

### PR DESCRIPTION
**Pull request by smith750**
Get GL unit tests running again; remove currently useless file enterprise feed unit test; add utility class to help us build batch directories for test

### CODE REVIEW

File: src/test/java/org/kuali/kfs/sys/batch/BatchDirectoryHelper.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    private boolean batchFileDirectoryCreated = false;
+
+    private volatile static ConfigurationService configurationService;
+
+    public BatchDirectoryHelper(String module, String directoryName) {
+        this.module = module;
+        this.directoryName = directoryName;
+    }
+
+    public void setUp() {
```
Comment (bhutchinson) [inline src/test/java/org/kuali/kfs/sys/batch/BatchDirectoryHelper.java]: do we want to call these methods setUp() and tearDown() which has special meaning to JUnit, even though this isn't a test class, or should they be renamed to something more descriptive: createBatchDirectory(), delete/removeBatchDirectory()?

Comment (smith750) [inline src/test/java/org/kuali/kfs/sys/batch/BatchDirectoryHelper.java]: Sure.

**Comment by smith750** [comment]
Hopefully better now...

**Comment by bhutchinson** [comment]
Thanks

**Pull request merged**